code-challenges.js

```diff
@@ -58,16 +30,6 @@ console.log(capIt(randomNouns))
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
-// create a function called orgNumOnly
-orgNumOnly = (arr) => {
-// map method for finding num only (typeof method)
-    let orgArr= arr.filter(el => typeof el === "number" )
-// .sort(el => (a, b) {return a-b}) to organize numbers
-    return orgArr.sort((a, b)=> a-b)
```

Comment:
great! you can even put this all in one line if you would like
ex: `return arr.filter(el => typeof el === "number" ).sort((a, b)=> a-b)`